src/cascadia/TerminalSettingsEditor/MainPage.xaml

```diff
@@ -10,54 +10,87 @@ the MIT License. See LICENSE in the project root for license information. -->
     xmlns:muxc="using:Microsoft.UI.Xaml.Controls"
     mc:Ignorable="d">
 
+    <Page.Resources>
+        <ResourceDictionary>
+            <ResourceDictionary.ThemeDictionaries>
+                <ResourceDictionary x:Key="Dark">
+                    <Thickness x:Key="NavigationViewHeaderMargin">20,5,0,0</Thickness>
+                </ResourceDictionary>
+
+                <ResourceDictionary x:Key="Light">
+                    <Thickness x:Key="NavigationViewHeaderMargin">20,5,0,0</Thickness>
+                </ResourceDictionary>
+
+                <ResourceDictionary x:Key="HighContrast">
+                    <Thickness x:Key="NavigationViewHeaderMargin">20,5,0,0</Thickness>
+                </ResourceDictionary>
+            </ResourceDictionary.ThemeDictionaries>
+        </ResourceDictionary>
+    </Page.Resources>
+
     <Grid>
         <Grid.Resources>
             <AcrylicBrush x:Key="NavigationViewDefaultPaneBackground"
-                  BackgroundSource="Backdrop" TintColor="{ThemeResource SystemChromeMediumColor}" TintOpacity=".5"/>
+                          BackgroundSource="Backdrop"
+                          TintColor="{ThemeResource SystemChromeMediumColor}"
+                          TintOpacity=".5"/>
             <AcrylicBrush x:Key="NavigationViewTopPaneBackground"
-                  BackgroundSource="Backdrop" TintColor="{ThemeResource SystemChromeMediumColor}" TintOpacity=".5"/>
+                          BackgroundSource="Backdrop"
+                          TintColor="{ThemeResource SystemChromeMediumColor}"
+                          TintOpacity=".5"/>
             <AcrylicBrush x:Key="NavigationViewExpandedPaneBackground"
-                  BackgroundSource="HostBackdrop" TintColor="{ThemeResource SystemChromeMediumColor}" TintOpacity=".7"/>
+                          BackgroundSource="HostBackdrop"
+                          TintColor="{ThemeResource SystemChromeMediumColor}"
+                          TintOpacity=".7"/>
         </Grid.Resources>
+
         <muxc:NavigationView x:Name="SettingsNav"
-                        IsSettingsVisible="False"
-                        Loaded="SettingsNav_Loaded"
-                        ItemInvoked="SettingsNav_ItemInvoked"
-                        IsBackButtonVisible="Auto"
-                        IsBackEnabled="True"
-                        BackRequested="SettingsNav_BackRequested">
+                             IsSettingsVisible="False"
+                             Header="{Binding ElementName=SettingsNav, Path=SelectedItem.Content, Mode=OneWay}"
+                             Loaded="SettingsNav_Loaded"
+                             ItemInvoked="SettingsNav_ItemInvoked"
+                             IsBackButtonVisible="Collapsed">
+
+            <muxc:NavigationView.HeaderTemplate>
+                <DataTemplate x:DataType="x:String">
+                    <TextBlock Text="{x:Bind}"
+                               Margin="0,10,0,0"

```

Comment:
Yeah. @cinnamon-msft was saying that the styling was designed for full-paged applications. Since we're in a tabbed application, worked with her to make it a bit smaller and _feel better_.

Reply:
We mainly actually just need this for the `Margin`, I think. I'll play around with the resource dictionary again to see if I can get rid of this data template.

Reply:
Wait no, I need it for the `TitleTextBlockStyle`. Could I get some help on applying that? I think you said there was a way to overwrite the `NavigationViewTitleHeaderContentControlTextStyle` with this in the resource dictionary?

Reply:
Begrudgingly, I accept that this is likely the correct way to do this.

Please make sure you figure out how the two margins interact and _get rid of one of them_.